### ISSUE TITLE
Autocomplete: options in result list is not selectable by ctrl+opt+space

### ISSUE THREAD

**Issue by SparrowSpit** [CMSgov/design-system#578]
**Describe the bug**
We saw in our (MCT) accessibility user-testing that a screenreader/keyboard user (using VoiceOver on Safari) could not use ctrl+opt+space to select an option from the results list in the Autocomplete component.  

**To Reproduce**
1.  place cursor in Autocomplete component https://design.cms.gov/components/autocomplete/
2. Type c then use ARROW keys to change options
3. try to select the option in the results list using ctrl+option+space
4. only selecting ENTER will select the option and enter it into the input

**Expected behavior**
We  need to ensure that the options are selectable with ctrl+option+space. Apple's documentation states that this combo of keys should "Perform the default action for the item in the VoiceOver cursor". 

Here is that documentation page: in the "Action" section, and read the VO+space bar action (the default VO keys are ctrl+space) https://www.apple.com/voiceover/info/guide/_1131.html .

 - Device: macbook
 - Browser: safari
 - Screenreader: VoiceOver

**Comment by davidakennedy** [comment]
Looking into this, I can't seem to reproduce it.

I'm using MacOS 10.14.6 with Safari 13.0.4. I'll see if I can get some others to try.

**Comment by bernardwang** [comment]
Closing from conversation in Slack, sounds like its from the MCT autocomplete implementation
